### PR TITLE
fixing problems related Arabic language direction in home page

### PR DESCRIPTION
**Pull request by hiba-machfej**
Screen Shot 2020-11-22 at 4 41 47 PM
Screen Shot 2020-11-22 at 4 41 52 PM
Screen Shot 2020-11-22 at 4 41 57 PM
Screen Shot 2020-11-22 at 4 42 06 PM
Screen Shot 2020-11-22 at 4 42 09 PM
Screen Shot 2020-11-22 at 4 42 13 PM

### CODE REVIEW

File: public/locales/ar/translation.json [hunk 1]
```diff
@@ -166,8 +166,9 @@
       }
     ],
     "buttons": {
-      "opportunitiesBtn": "جميع الفرص",
-      "blogBtn": "جميع المدونات"
+      "opportunitiesBtn": "جميع الفرص {{arrow}}",
+      "blogBtn": "جميع المدونات {{arrow}} ",
+      "arrow": "&#8592;"
```
Comment (louisrli) [inline public/locales/ar/translation.json]: i dont think you have to do this arrow thing. if you just put it inline in the html it should handle RTL correctly next to it

Comment (louisrli) [reply public/locales/ar/translation.json]: i mean arrow does not have to be in the translations

the word "arrow" is not being translated, therefore it doesnt make sense to put them in translations

again think from the perspective of a translator that receives these files

they would just be like: "wtf theres an arrow thing here."

Comment (hiba-machfej) [reply public/locales/ar/translation.json]: correct but &#8592; and &#8594; are different and they are not being flipped.

Comment (hiba-machfej) [reply public/locales/ar/translation.json]: I'll try to fix the issue

**Comment by louisrli** [comment]
you should be able to do something like "if rtl then do < else do >"

**Comment by hiba-machfej** [comment]
I added if condition 
Screen Shot 2020-11-22 at 6 57 52 PM
Screen Shot 2020-11-22 at 6 58 00 PM

File: public/locales/ar/translation.json [hunk 2]
```diff
@@ -166,8 +166,8 @@
       }
     ],
     "buttons": {
-      "opportunitiesBtn": "جميع الفرص",
-      "blogBtn": "جميع المدونات"
+      "opportunitiesBtn": "جميع الفرص {{arrow}}",
```
Comment (louisrli) [inline public/locales/ar/translation.json]: you shouldnt need this here though. just append it

{t(opportunitiesBtn)} {arrow} in the html

File: src/components/JobsTrainingHome/index.jsx [hunk 3]
```diff
@@ -35,9 +35,16 @@ export default function JobsTrainingHome() {
           </Col>
         </Row>
         <Link to="/opportunities">
-          <h6 className="opportunitiesButton">
-            {t("home.buttons.opportunitiesBtn")} <span>&#8594;</span>
-          </h6>
+          <Row className="opportunitiesButton">
+            <h6>
+              {t("home.buttons.opportunitiesBtn")}
```
Comment (louisrli) [inline src/components/JobsTrainingHome/index.jsx:40]: yeah it looks like you forgot to delete the arrow thing

Comment (hiba-machfej) [reply src/components/JobsTrainingHome/index.jsx:40]: sorry I guess I am bit drunk from all the coding :/

Comment (hiba-machfej) [reply src/components/JobsTrainingHome/index.jsx:40]: changed it

File: src/components/Navbar/index.scss [hunk 4]
```diff
@@ -7,8 +7,10 @@
   height: 5rem;
   font-family: $mainFontFamily;
 
+  #responsive-navbar-nav {
```
Comment (louisrli) [inline src/components/Navbar/index.scss:11]: is this a bootstrap class or something? add a comment above why its using ids

**Review by louisrli — APPROVED** [review]

**Review by jennetturkkan — APPROVED** [review]

**Comment by hiba-machfej** [comment]
@Ammar-64

**Pull request merged**